docs/firmware/3-zephyr-sdk/2-samples/dfu.md

```diff
@@ -29,10 +29,9 @@ $ mcumgr --conntype serial --connstring /dev/ttyUSB,baudrate=1000000 build/zephy
 Now rebuild application with assigned new version to 1.2.3 to
 distinguish it from old firmware:
 
-    .. code-block:: console
-
-> \$ west build -p -b circuitdojo_feather_nrf9160ns samples/dfu \--
-> -DCONFIG_MCUBOOT_IMAGE_VERSION=\"1.2.3\"
+``` {.console}
+$ west build -p -b circuitdojo_feather_nrf9160ns samples/dfu -- -DCONFIG_MCUBOOT_IMAGE_VERSION=\"1.2.3\"
+```
```

Comment:
It is much easier to follow git history if actual changes (like adding ` -p -- -DCONFIG_MCUBOOT_IMAGE_VERSION=\"1.0.1\"` above) are separated from reformatting of current code/docs.

Reply:
Agreed. Separate content changes from structure changes where possible.